cli/fossilize_replay.cpp

```diff
@@ -231,6 +231,8 @@ struct ThreadedReplayer : StateCreatorInterface
 
 		unsigned shader_cache_size_mb = 256;
 
+		const char *pipeline_hash = nullptr;
```

Comment:
This should be std::string, and not part of the ThreadedReplayer::Options since it does not affect the ThreadedReplayer.

Reply:
Alternatively, the Hash uint64_t can be parsed directly when parsing arguments, no need to pass around a string.

Reply:
Looking at it some more, just keep the pipeline_hash option here, and change the type to Hash directly.

Reply:
> Alternatively, the Hash uint64_t can be parsed directly when parsing arguments, no need to pass around a string.

I thought about that but I couldn't figure out how to properly propagate the error from the parser lambda.  I guess I could call exit()?

Reply:
Log and exit() is fine with me.